public/app/plugins/datasource/prometheus/querybuilder/components/promQail/state/helpers.ts

```diff
@@ -158,10 +281,26 @@ export async function promQailSuggest(
 
   const interactionToUpdate = interaction ? interaction : createInteraction(SuggestionType.Historical);
 
+  // Decide metric type
+  await datasource.languageProvider.start(); //why do I need to do this to prepropulate the list??
```

Comment:
If you only need metadata you can do this instead of `.start()`

https://github.com/grafana/grafana/blob/a99485ba331b716af953f02bec4b3325761734f6/public/app/plugins/datasource/prometheus/querybuilder/components/MetricsLabelsSection.tsx#L235C1-L244C4